index.html

```diff
@@ -295,6 +296,15 @@ <h1 class="brand-logo center">Labyrinth</h1>
 
   <!-- Game Tiles  -->
   <div id="tiles"></div>
-
+  <!-- Caret-up-down -->
+  <script>
+    var $z=$('.select-your-language>a');
+    var $a=$('.languagepicker.roundborders.large')
+      $a.hover(function(){
+        $z.html('Select your language <span class="fa fa-caret-up"></span>');
+      },function(){
+        $z.html('Select your language <span class="fa fa-caret-down"></span>');
+      })
+  </script>
```

Comment:
Thanks!
I think, you can do this with the :hover css property. Would you like to try it?
Also, adding Javascript at the end of the file is not the style we go for. For consistency, that would go to the js folder if it is still needed.

Reply:
there is also :after to add text.

Reply:
@niccokunzmann I'll add the javascript to sidebar.js 
Is that fine?

Reply:
You can do that, sounds ok. If the DOM is not loaded when the script is executed,
you can use `document.addEventListener("load", function (){ your code goes here})`.

Reply:
looks like emails can not reply to these comments.

Reply:
Okay @niccokunzmann I'll make the changes